providers/src/airflow/providers/edge/provider.yaml

```diff
@@ -102,3 +102,11 @@ config:
         type: integer
         example: ~
         default: "60"
+      push_log_chunk_size:
+        description: |
+          Edge Worker uploads log files in chunks. If the log file part which is uploaded exceeds the chunk size it creates a new request. This solves issue if firewall defines max size of request.
+          This value must be defined in Bytes.
+        version_added: ~
+        type: integer
+        example: ~
+        default: "1024"
```

Comment:
Following the NGinx documentation in https://nginx.org/en/docs/http/ngx_http_core_module.html#client_max_body_size the default is 1MB.

I'd propose to set the default to 512KB

Reply:
Done